mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -3883,6 +3883,55 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 				log.Printf("[INFO] GKE cluster %s: default-pool setting for insecure_kubelet_readonly_port_enabled updated to %s", d.Id(), it)
       }
 		}
+
+		if d.HasChange("node_config.0.gcfs_config") {
```

Comment:
This kind of takes us in the opposite direction as far as DRYing it up for https://github.com/hashicorp/terraform-provider-google/issues/19225, and might also cause a few conflicts with #11697, but this is probably the most expedient fix, and I don't think will take us _too_ far in the opposite direction.